src/Bridge/Symfony/XliffFileDumper.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Translation\PlatformAdapter\PhraseApp\Bridge\Symfony;
+
+use Symfony\Component\Translation\Dumper\FileDumper;
+use Symfony\Component\Translation\MessageCatalogue;
+
+/**
+ * Custom XliffFileDumper for use with phrase app
+ */
+class XliffFileDumper extends FileDumper
```

Comment:
It seams wrong to add 3 classes to change 2 lines of code. 

Couldn't we find another solution, example to put the domains in a tag?